CSS property: scroll-initial-target (#43155)
mdn/content: files/en-us/web/css/reference/properties/scroll-initial-target/index.md

---
title: scroll-initial-target
slug: Web/CSS/Reference/Properties/scroll-initial-target
page-type: css-property
browser-compat: css.properties.scroll-initial-target
sidebar: cssref
---

The **`scroll-initial-target`** [CSS](/en-US/docs/Web/CSS) property enables defining elements that should be snapped when scroll containers are first rendered.

## Syntax

```css
/* Keyword values */
scroll-initial-target: none;
scroll-initial-target: nearest;

/* Global values */
scroll-initial-target: inherit;
scroll-initial-target: initial;
scroll-initial-target: revert;
scroll-initial-target: revert-layer;
scroll-initial-target: unset;
```

### Values

- `none`
  - : The element is not an initial scroll target.
- `nearest`
  - : The element is potentially an initial scroll target for its nearest scroll container ancestor.

## Description

> The `scroll-initial-target` property enables defining elements that should be snapped when scroll snap containers are first rendered. Setting the value to `nearest` defines the element as a potential target that should be snapped when the nearest ancestor {{glossary("scroll container")}} what that container first shows up on the page.

Reviewer: As above for first sentence.

Reviewer: Second sentence is broken - see "what that container".

What I think this means is (something like)

```suggestion
The `scroll-initial-target` property allows you to define elements that should be snapped when scroll snap containers are first rendered. Setting the value to `nearest` marks the nearest ancestor {{glossary("scroll container")}} as a potential target that should be snapped when it (the container) first shows up on the page.
```
